src/CMakeLists.txt

```diff
@@ -2,15 +2,19 @@
 
 if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
   set(CMAKE_Fortran_FLAGS
-      "-g -xHOST -traceback -free -convert big_endian -assume byterecl")
+      "-g -xHOST -traceback -free -convert big_endian -assume byterecl ${CMAKE_Fortran_FLAGS}")
```

Comment:
is `-xHOST` required here?

Reply:
Please remove or replace with
```
-axSSE4.2,AVX,CORE-AVX2,CORE-AVX512
```
or
```
-axSSE4.2,AVX,CORE-AVX2
```

@DusanJovic-NOAA any preference on adding AVX512 or not?

Reply:
I notice that it's all the I/O libraries that have `-xHost` (sigio, sfcio, gfsio). Is there a reason for that I'm missing? I would think vectorization and other optimizations would have little effect on a library like this.

Reply:
I think we should not hard code any architecture specific flags.